torch_geometric/nn/conv/cugraph/base.py

```diff
@@ -128,7 +137,7 @@ def get_typed_cugraph(
         if num_edge_types is None:
             num_edge_types = int(edge_type.max()) + 1
 
-        row, colptr = csc
+        row, colptr, num_src_nodes = csc
         edge_type = edge_type.int()
```

Comment:
similar to above, add a comment (simply refer to the comment in the above function)